src/app/contextBroker/contextBroker.cpp

```diff
@@ -291,12 +292,12 @@ unsigned int    reqPoolSize;
 #define WRITE_CONCERN_DESC     "db write concern (0:unacknowledged, 1:acknowledged)"
 #define CPR_FORWARD_LIMIT_DESC "maximum number of forwarded requests to Context Providers for a single client request"
 #define SUB_CACHE_IVAL_DESC    "interval in seconds between calls to Subscription Cache refresh (0: no refresh)"
-#define NOTIFICATION_MODE_DESC "notification mode (persistent|transient|none)"
+#define NOTIFICATION_MODE_DESC "notification mode (persistent|transient|threadpool:q:n)"
 #define NO_CACHE               "disable subscription cache for lookups"
 #define CONN_MEMORY_DESC       "maximum memory size per connection (in kilobytes)"
 #define MAX_CONN_DESC          "maximum number of simultaneous connections"
 #define REQ_POOL_SIZE          "size of thread pool for incoming connections"
-
+#define DROP_NOTIF_DESC        "drop outgoing notifications (testing)"
```

Comment:
Suggestion:

DROP_NOTIF_DESC -> SIMULATED_NOTIF_DESC
Text: "simulate notificaitons instead of actual sending them (only for testing)"